src/dropin-model.js

```diff
@@ -24,6 +25,10 @@ DropinModel.prototype = Object.create(EventEmitter.prototype, {
   constructor: DropinModel
 });
 
+DropinModel.prototype.isPaymentMethodRequestable = function () {
+  return Boolean(this._paymentMethodIsRequestable);
```

Comment:
We could do `this._paymentMethodIsRequestable` as long as we initialize `this._paymentMethidIsRequestable` as false. I just like the explicit nature of always returning a `Boolean` in this method.